src/library.h

```diff
@@ -112,6 +112,7 @@ void  lammps_commands_string(void *handle, const char *str);
  * ----------------------------------------------------------------------- */
 
 double lammps_get_natoms(void *handle);
+int lammps_get_nbonds(void *handle);
```

Comment:
Should be `double` to be consistent.

Reply:
I have removed `get_nbonds()` and used `get_thermo('bonds')` in the example to get the number of bonds that make this function obsolete.